docs/how_to/single_node.md

```diff
@@ -14,8 +14,8 @@ To begin, first start up a Docker container with port `7201` (used to manage the
 directory on your host for durability:
 
 ```
-docker pull quay.io/m3/m3dbnode:latest
-docker run -p 7201:7201 -p 7203:7203 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db -v <PATH_TO_M3DB_CONFIG.yml>:/etc/m3dbnode/m3dbnode.yml quay.io/m3/m3dbnode:latest
+docker pull quay.io/m3db/m3dbnode:latest
```

Comment:
Eh?  Why are we changing the Quay repo people are using?

Have all the changes you been making for the :latest tag, etc been to the other Quay repo?

Reply:
It looks like we disabled builds for the other one, or that one only publishes tags and not latest `master`. I thought we wanted to deprecate it or something and only have our builds published to the same repo as our GH name, do we have a plan for that?